styles/config.css

```diff
@@ -114,9 +113,24 @@ body {
   text-decoration: none;
 }
 
+.warp-coper{

```

Comment:
@vnxx can we use some meaningful name here?

Reply:
Done, I change it to 'wrapper-cover'